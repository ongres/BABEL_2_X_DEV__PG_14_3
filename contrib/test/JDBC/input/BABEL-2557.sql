select cast(hashbytes('sha2_512', 'abcdefg') as varbinary(64))
GO

select cast(hashbytes('sha2_512', 'abcdefg') as varbinary(62))
GO