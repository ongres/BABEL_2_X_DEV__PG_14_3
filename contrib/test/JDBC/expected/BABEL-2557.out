select cast(hashbytes('sha2_512', 'abcdefg') as varbinary(64))
GO
~~START~~
varbinary
D716A4188569B68AB1B6DFAC178E570114CDF0EA3A1CC0E31486C3E41241BC6A76424E8C37AB26F096FC85EF9886C8CB634187F4FDDFF645FB099F1FF54C6B8C
~~END~~


select cast(hashbytes('sha2_512', 'abcdefg') as varbinary(62))
GO
~~START~~
varbinary
D716A4188569B68AB1B6DFAC178E570114CDF0EA3A1CC0E31486C3E41241BC6A76424E8C37AB26F096FC85EF9886C8CB634187F4FDDFF645FB099F1FF54C
~~END~~

